src/scss/components/_scenarios.scss

```diff
@@ -0,0 +1,106 @@
+.scenarios {
+  $main: &;
+
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 230px;
+  padding-top: 30px;
+  position: relative;
+  width: 100%;
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    height: 54px;
+    line-height: 30px;
+    margin-bottom: 102px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    width: 227px;
+  }
+
+  &__principal-container {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    margin: auto;
+    padding-left: 65PX;
+    padding-right: 63PX;
+    position: relative;
+  }
+
+  &__container-images {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
+
+  &__text {
+    font-family: $hkgrotesk-bold;
+    font-size: 18px;
+    margin-bottom: 20px;
+    margin-top: 20px;
+  }
+
+  &__images--0 {
+    display: flex;
+    height: auto;
+    position: absolute;
+    right: 32px;
+    text-align: right;
+    top: -89px;
+    width: 40%;
+    z-index: 1;
+  }
+
+  &__images--1 {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    position: relative;
+    text-align: left;
+    width: 100%;
+
+    &::before {
+      background-image: url('../static/assets/forma-azul.png');
+      background-repeat: no-repeat;
+      background-size: 100% 100%;
+      content: '';
+      height: 26%;
+      left: 22%;
+      position: absolute;
+      top: 13px;
+      width: 48%;
+      z-index: -1;
+    }
+
+    &::after {
+      background-image: url('../static/assets/forma-puntos.png');
+      background-repeat: no-repeat;
+      background-size: 100% 100%;
+      bottom: -7%;
+      content: '';
+      height: 8%;
+      position: absolute;
+      right: -2px;
+      text-align: right;
+      width: 20%;
+    }
+  }
+
+  &__images--2 {
+    display: flex;
+    flex-direction: column-reverse;
+    height: auto;
```

Comment:
Here, remove `height: auto;` and `text-align: left;` because those are already default values for those properties

Reply:
Removed